numerics/fast_sin_cos_cycle.hpp

```diff
@@ -0,0 +1,11 @@
+﻿
+#pragma once
+
+namespace principia {
+namespace numerics {
+
+//TODO(phl):comment
+void FastSinCosCycle(double x, double& sin, double& cos);
```

Comment:
SinCos2π(double cycles, etc.)?

cf. the IEEE 754-2008 sinPi etc.

Reply:
Done.